cmake/modules/AddSwift.cmake

```diff
@@ -239,6 +239,10 @@ function(_add_variant_c_compile_flags)
     list(APPEND result "-DNDEBUG")
   endif()
 
+  if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
+    list(APPEND result "-Wno-inconsistent-missing-override")
```

Comment:
I think we actually just want to fix these in most cases. Those particular files really are different from the others.